packages/flutter/test/widgets/scroll_view_test.dart

```diff
@@ -128,6 +128,38 @@ void main() {
     expect(textField.focusNode.hasFocus, isFalse);
   });
 
+  testWidgets('GridView dismiss keyboard onDrag test', (WidgetTester tester) async {
+    final List<FocusNode> focusNodes = List<FocusNode>.generate(50, (int i) => FocusNode());
+
+    await tester.pumpWidget(textFieldBoilerplate(
+        child: GridView.builder(
+      padding: const EdgeInsets.all(0),
+      gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount:2),
+      keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
+      itemCount: focusNodes.length,
+      itemBuilder: (BuildContext context, int index){
+        return Container(
+          height: 50,
+          color: Colors.green,
+          child: TextField(
+              focusNode: focusNodes[index],
+              style: const TextStyle(
+                fontSize: 24,
+                fontWeight: FontWeight.bold,
+              )));
+      },
+    )));
```

Comment:
nit: fix indentation:

```suggestion
      child: GridView.builder(
      padding: const EdgeInsets.all(0),
      gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount:2),
      keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
      itemCount: focusNodes.length,
      itemBuilder: (BuildContext context, int index){
        return Container(
          height: 50,
          color: Colors.green,
          child: TextField(
              focusNode: focusNodes[index],
              style: const TextStyle(
                fontSize: 24,
                fontWeight: FontWeight.bold,
              ),
            ),
          );
        },
      ),
    ));
```

Reply:
thanks for your review. I have some doubt about format.  I have tried to format file  by  flutter format xxx.dart ,  but it doesn't meet the requirements. Could you please tell me how to check it and what's the standard? thanks

Reply:
@Hixie Could you please clarify the rule of flutter format when you have a time? i have tried flutter format xxx.dart and   flutter analyze. but it seems not good.

Reply:
The rules for the style of this repository are written down in https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo.

Most notably, this repository does not use automatic formatting (e.g. with dartformat or `flutter format`). This rule about trailing commas is also relevant for this particular case: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line

Reply:
It's kind of you. Thank you for your patience. please take a look when you have a time.